### DISCUSSION TITLE
Is it possible to eavesdrop on a conversation?

### DISCUSSION THREAD

**Discussion by anonimus007-700** [discordjs/discord.js#9819] [Q&A] [answered]
### Which application or package is this feature request for?

discord.js

### Feature

I'm creating a bot for Discord that will join a voice channel and record everything that's being said there. I tried doing it with discord.py, but it doesn't have such a feature, so I decided to switch to discord.js. Could you please tell me what this feature is called?

### Ideal solution or implementation

In an ideal scenario, it would be great to describe it with code, but I would be very grateful even if you simply tell me the name of the function.

**Comment by almostSouji** [comment, marked as answer]
You can find a guide on using the voice library at https://discordjs.guide/voice/
Reminder, since your phrasing is... odd:
Listening in on people's conversations without explicit consent and being explicit about this functionality of your bot is a privacy and terms of service violation (referred to as "surprising features").

If your bot is made for this purpose, that's alright. Don't abuse your bots privileges to spy on people because you are bored or have questionable morals, though.
